package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@reportportal/agent-js-jest",
-  "version": "1.0.7",
+  "version": "5.0.0",
```

Comment:
Do not change this version manually. It will be extracted from auto-generated git tag.